src/components/EnterpriseList/index.jsx

```diff
@@ -53,7 +53,7 @@ class EnterpriseList extends React.Component {
   fetchEnterprisesWithSearch = (options) => {
     const optionsWithSearch = {
       ...options,
-      search: this.state.searchQuery || undefined,
+      ...(this.state.searchQuery && { search: this.state.searchQuery }),
```

Comment:
Makes sense to me, though, (optional nit) I wonder if the following is slightly more readable as:

```jsx
const optionsWithSearch = { ...options };
if (this.state.searchQuery) {
  optionsWithSearch.search = this.state.searchQuery;
}
```

Reply:
ah yeah I see now, this is returning undefined for the search value so simply avoiding it in the object is good.

optional:
if you wanted to harden the function fetchEnterprisesList in case of undefined, you can also do sth like this there (optionally, since this won't happen after your change. I just don't think functions should assume correctness of their inputs generally)

```
const { search, ...restopts} = options
const finalOpts = search ? { ...options } : restopts
```

or sth close to this

Reply:
done